### ISSUE TITLE
[MWRAPPER-84] Update documentation about shells

### ISSUE THREAD

**Issue by jira-importer** [apache/maven-wrapper#242]
**[Michael Osipov](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=michael-o)** opened **[MWRAPPER-84](https://issues.apache.org/jira/browse/MWRAPPER-84?redirect=false)** and commented

https://maven.apache.org/wrapper/ says:

```
For all those *nix operating systems, various shells should work including
    sh
    bash
    dash
    zsh
```

This section must be removed an replaced that the wrapper script required a POSIX-compatible Bourne shell.

---

**Affects:** 3.1.1

**Issue Links:**
- [MWRAPPER-83](https://issues.apache.org/jira/browse/MWRAPPER-83) mvnw fails in zsh due to empty MAVEN_OPTS variable
 (_**"is caused by"**_)

**Remote Links:**
- [GitHub Pull Request #70
](https://github.com/apache/maven-wrapper/pull/70)

**Comment by jira-importer** [comment]
**[Michael Osipov](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=michael-o)** commented

[Scott Kurz](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=scottkurz), PR created to clean up the failures with the zsh.

**Comment by jira-importer** [comment]
**[Michael Osipov](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=michael-o)** commented

Fixed with  [aaa1de7f2e4f233261de92923a1c0e1ae322a02b](https://gitbox.apache.org/repos/asf?p=maven-wrapper.git;a=commit;h=aaa1de7f2e4f233261de92923a1c0e1ae322a02b).
